tronbyt_server/templates/auth/edit.html

```diff
@@ -34,4 +34,11 @@ <h1>{{ _('Edit Password') }}</h1>
     <input type="submit" value="{{ _('Save') }}" class="w3-button w3-green"></input>
 </form>
 
+<h1>{{ _('Edit API Key') }}</h1>
```

Comment:
Could you please add "API-Schlüssel bearbeiten" as the German translation for this key?